input/system/azure/logs.go

```diff
@@ -251,7 +251,7 @@ func ParseRecordToLogLines(in AzurePostgresLogRecord) ([]state.LogLine, string,
 		azureDbServerName = in.LogicalServerName
 	}
 
-	logLine, ok := logs.ParseLogLineWithPrefix("", logLineContent)
+	logLine, ok := logs.ParseLogLineWithPrefix("", logLineContent, nil)
```

Comment:
Systems where we collect logs for multiple servers through the same stream are harder to fix (but on the plus side, far less likely to have this problem).

Reply:
Looking at this again, this is not impossible: we'd need to pass in a mapping of server identifier (as it occurs in the log stream) to *time.Location. I'll hold off on addressing this until we decide if this is a reasonable and worthwhile approach.